policies/verify-instances-templates-reference/policies/rego_rule.rego

```diff
@@ -1,33 +1,32 @@
-
 package crownlabs_instance_template_reference
 
-t_ns := input.review.object.spec["template.crownlabs.polito.it/TemplateRef"].namespace
+tmpl_api_version := "crownlabs.polito.it/v1alpha2"
 
-ns = t_ns {
-	count([t_ns]) > 0
-	
-}
+tmpl_ref_field := "template.crownlabs.polito.it/TemplateRef"
 
-else = "default" {
-	true
+# Return the referenced template namespace, in case the field is defined
+get_tmpl_namespace(review) = namespace {
+	review.object.spec[tmpl_ref_field].namespace
+	namespace := input.review.object.spec[tmpl_ref_field].namespace
```

Comment:
Since the function receives "review" as input parameter, it could be directly used instead of "input.review". Anyways, the result should be the same.

Reply:
Fixed, thanks!